azure-pipelines.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           go install github.com/jstemmer/go-junit-report@v0.9.1
           go install github.com/axw/gocov/gocov@v1.1.0
           go install github.com/AlekSi/gocov-xml@v1.0.0
-          go install github.com/matm/gocov-html@v0.0.0-20200509184451-71874e2e203b
```

Comment:
We should probably still lock to a version, but I've got no immediate objection to this release.

Reply:
Good point, fixed.